frontend/src/components/UserDetail/UserDetail.tsx

```diff
@@ -111,9 +111,10 @@ const UserDetail = ({
           </div>
           <div className={styles.userEditContainer}>
             {rider && !isRider ? (
-              <Button onClick={toggleActive}>
-                {rider.active ? 'Deactivate' : 'Activate'}
-              </Button>
+              <label className={styles.switch}>
+                <input type="checkbox" />
```

Comment:
We want to be able to control the button in-code as well  - which means we want to have a variable which when true, will set the button to be on when the user loads the page.  We don't want the button to default to false whenever the user opens the page (because we want it to match the state of the user which could be on or off at the time the user visits this page)